devito/ir/iet/algorithms.py

```diff
@@ -0,0 +1,45 @@
+from collections import OrderedDict
```

Comment:
note @mloubout : this file previously was https://github.com/devitocodes/devito/blob/master/devito/ir/iet/scheduler.py

I changed its name since I finally managed to remove the iet_lower* routines (which were disgusting -- performing that sort of Dimension lowering when we already have an IET... ).